src/style.css

```
@@ -1 +1,75 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+:root {
+  --active-hover-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background: #fff;
+  justify-content: flex-end;
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  padding: 10px 0;
+}
+
+.header_nav {
```

```suggestion
.header__nav {
```
now these style are not applied